task/servicetest/servicetest.go

```diff
@@ -937,7 +936,7 @@ func testTaskConcurrency(t *testing.T, sys *System) {
 			// Create a run for the last task we found.
 			// The script should run every minute, so use max now.
 			tid := tasks[len(tasks)-1].ID
-			if _, err := sys.TaskControlService.CreateNextRun(sys.Ctx, tid, math.MaxInt64); err != nil {
+			if _, err := sys.TaskControlService.CreateNextRun(sys.Ctx, tid, math.MaxInt64>>6); err != nil {
```

Comment:
Is `>>6` needed here?

Reply:
Yes, because math.MaxInt64 is too large and causes errors when converting to a time and then adding durations etc.